examples/utils/example-utils/src/migrationTool/sameContainerMigrationTool.ts

```diff
@@ -32,6 +40,32 @@ export class SameContainerMigrationTool extends DataObject implements ISameConta
 		return this._setContainerRef;
 	}
 
+	private _setLoadPausedContainerFn:
+		| ((fn: (sequenceNumber: number) => Promise<ISameContainerMigratableModel>) => void)
+		| undefined;
+	public get setLoadPausedContainerFn(): (
+		fn: (sequenceNumber: number) => Promise<ISameContainerMigratableModel>,
+	) => void {
+		if (this._setLoadPausedContainerFn === undefined) {
+			throw new Error("_setloadPausedContainerFn did not initialize properly");
+		}
+		return this._setLoadPausedContainerFn;
+	}
+
+	/**
+	 * The sequence number that the proposal was accepted at, or undefined if it was not yet accepted.
+	 */
+	private _acceptedSeqNum: number | undefined;
+	/**
+	 * The sequence number that the proposal was accepted at.
+	 */
+	private get acceptedSeqNum(): number {
```

Comment:
The Migrator would then need access to this to do the load - so should not throw and should return `number | undefined` but with a strong contract for what `SameContainerMigrationState` it should be defined.